basic_hashtable/b_hashtables.c

```diff
@@ -94,7 +102,9 @@ void hash_table_insert(BasicHashTable *ht, char *key, char *value)
  ****/
 void hash_table_remove(BasicHashTable *ht, char *key)
 {
-
+  unsigned int index = hash(key, ht->capacity);
+  ht->storage[index] = NULL;
+  destroy_pair(ht->storage[index]);
```

Comment:
Great job using destroy_pair, ht->storage[index] but you can still have junk.
Try ht->storage[index] = NULL;